src/main/java/gregtech/api/cover/ICoverableTile.java

```diff
@@ -0,0 +1,65 @@
+package gregtech.api.cover;
+
+import gregtech.api.capability.ICoverHandler;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public interface ICoverableTile extends ICapabilityProvider {
+
+    World getWorld();
```

Comment:
Will break in obfuscated environment and crash with NoSuchMethodError, just like getPos(). Name methods in a way different from vanilla MC.

Reply:
Yep, I know, and this is fixed in commit 637ce6e59b5be

Reply:
you should push all things in one PR, that's the point. Otherwise it won't merge properly.

Reply:
Hmmm... You mean I should put those commits in this pr again?